clusterman/interfaces/cluster_connector.py

```diff
@@ -46,6 +46,14 @@ def get_agent_metadata(self, ip_address: Optional[str]) -> AgentMetadata:
             return AgentMetadata()
         return self._get_agent_metadata(ip_address)
 
+    def freeze_agent(self, agent_id: Optional[str]) -> None:
```

Comment:
why do we need a wrapper method here? if `agent_id` is always required, can we just make it non optional?